jujugui/static/gui/src/app/components/deployment-flow/sshkey/sshkey.js

```diff
@@ -55,50 +350,55 @@ class DeploymentSSHKey extends React.Component {
 
     let message = (
       <p>
-        Optionally provide a SSH key (e.g. ~/.ssh/id_rsa.pub) to allow
-        accessing machines provisioned on this model via "juju ssh".
-        <br/> SSH keys can be added at any time using "juju add-ssh-key" or
-        "juju import-ssh-key".
+        Keys will allow you SSH access to the machines provisioned by Juju
```

Comment:
Reminded that we should talk with @elvirafigueras about copy here if we haven't already (also, sorry I didn't respond to your email, @elvirafigueras!)